.github/workflows/recipe-catalog-change.yaml

```diff
@@ -0,0 +1,222 @@
+name: Run recipe tests on catalog change
+
+on:
+  push:
+    paths:
+      - 'packages/backend/src/assets/ai.json'
+  pull_request:
+    paths:
+      - 'packages/backend/src/assets/ai.json'
+
+jobs:
+  windows:
+    name: windows-${{ matrix.windows-version }}-${{ matrix.windows-featurepack }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      checks: write
+    env:
+      MAPT_VERSION: v0.6.8
```

Comment:
You can use newer version of mapt. https://github.com/podman-desktop/extension-minikube/blob/main/.github/workflows/e2e-windows.yaml#L78

Reply:
fixed